README.md

```diff
@@ -20,3 +20,7 @@ json files somewhere else.
 
 In info-beamer just input the path to the cached json files. Authentication
 is not supported as of now.
+
+## Emoji
+
+Iconography from [Mutant Standard Emoji](https://mutant.tech).
```

Comment:
This should be in the COPYRIGHT file so it gets displayed on the correct page on info-beamer.com

Reply:
Fixed.